diff --git a/hw/scripts/shell.tcl.in b/hw/scripts/shell.tcl.in
index a9302ac..f096f2a 100644
--- a/hw/scripts/shell.tcl.in
+++ b/hw/scripts/shell.tcl.in
@@ -56,10 +56,12 @@ for {set i 0}  {$i < $cfg(n_config)} {incr i} {
 }
 
 # Call write HDL scripts
-unset ::env(PYTHONPATH)
-unset ::env(PYTHONHOME)
+# unset commands are removed (it doesn't work when jinja2 is installed with nix script)
+# unset ::env(PYTHONPATH)
+# unset ::env(PYTHONHOME)
 proc call_write_hdl {r_path} {
-    set output [exec /usr/bin/python3 "$r_path/write_hdl.py" 0 0]
+    # set output [exec /usr/bin/python3 "$r_path/write_hdl.py" 0 0]
+    set output [exec python3 "$r_path/write_hdl.py" 0 0]
     puts $output
 }
 call_write_hdl $build_dir
